src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import 'url(https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap)';
```

Comment:
There is a syntax error in the import statement for Google Fonts. It should be corrected to `@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');` to properly import the font.